src/topotoolbox/gridmixins/fillsinks.py

```diff
@@ -0,0 +1,34 @@
+import numpy as np
+import ctypes
+import os
+import copy
+
+class FillsinksMixin():
+    def fillsinks(self, maxdepth=None, sinks=None):
+
+        #TODO: arguments need to be evaluated.
+
+        basedir = os.path.dirname(__file__)
+        parentdir = os.path.dirname(basedir)
+        libpath = os.path.join(parentdir, 'libtopotoolbox.so')
```

Comment:
This could be solved by  using something like `sys.platform` and appending the different library ending for each os. I assume it would be `libtopotoolbox.dll`right?

Reply:
Windows leaves off the "lib" too, so it is `topotoolbox.dll`. macOS uses `libtopotoolbox.dylib` for its shared library.